numpy/_core/tests/test_arrayprint.py

```diff
@@ -533,6 +533,7 @@ def test_nested_array_repr(self):
         )
 
     @given(hynp.from_dtype(np.dtype("U")))
+    @pytest.mark.thread_unsafe(reason="gives unreliable results w/ hypothesis")
```

Comment:
Hypothesis is supposed to be thread-safe. I'd like to understand this a little better.